CIP-????/README.md

```diff
@@ -0,0 +1,85 @@
+--
+CIP: ?
+Title: Explicit script return values
```

Comment:
PR retitled same as CIP (and on meeting agenda # 81) in keeping with most of them being noun phrases rather than verbs made to look like nouns.  If you think `Checking` is more clear please reset both titles that way or let me know & I'll do it.

Reply:
Seems to be airing well with new name, so resolving